nl-NL.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+"This option makes the jailbreak dynamically set the CS_DEBUGGED codesign flag for every new process." = "Deze optie zorgt ervoor dat de jailbreak dynamisch de CS_DEBUGGED codesign instelt voor elk nieuw proces."
+"This option makes the kernel allow processes to run with invalid executable pages." = "Deze optie zorgt ervoor dat de kernel processen uit kan voeren met ongeldige uitvoerbare pagina's."
+"Auto Respring" = "Automatische Respring"
+"This option makes the jailbreak automatically restart the SpringBoard as soon as the jailbreak process is completed without the confirmation." = "Met deze optie start de jailbreak automatisch het SpringBoard zodra het jailbreakproces zonder bevestiging is voltooid."
+"iOS 11.0-12.1.2 on arm64 SoCs A7-A11." = "iOS 11.0-12.1.2 op arm64 SoCs A7-A11."
+"Read-Only RootFS" = "RootFS alleen lezen"
+"This option makes the jailbreak skip remounting the root filesystem as read-write when combined with the SSH-Only option." = "Deze optie zorgt ervoor dat de jailbreak het remounten van het rootbestandssysteem overslaat als lezen-schrijven in combinatie met de optie 'Alleen SSH'."
+"iOS 11.0-12.1.2 on arm64 SoCs A7-A12." = "iOS 11.0-12.1.2 op arm64 SoCs A7-A12."
+"Jailbreaking" = "Aan het jailbreaken"
+"Unable to verify my user id." = "Kan mijn gebruikers-ID niet verifiëren."
+"Memory allocation failure: %s" = "Geheugentoewijzingsfout: %s"
+"Unable to clean success file." = "Kan het succesbestand niet opschonen."
+"Unable to synchronize app preferences. Please restart the app and try again." = "Kan de voorkeuren niet synchroniseren. Start de applicatie opnieuw op en probeer het opnieuw."
+"Unable to create test file." = "Kan geen testbestand maken."
+"Unable to clean test file." = "Kan het testbestand niet opschonen."
+"Unable to inject trust cache" = "Kan de trust cache niet injecteren."
+"Exploiting kernel..." = "Kernel exploiteren..."
+"Unable to get host port." = "Kan hostpoort niet verkrijgen."
+"No exploit selected." = "Geen exploit geselecteerd."
+"Unable to exploit kernel. This is not an error. Reboot and try again." = "Kan kernel niet exploiteren. Dit is geen fout. Start de applicatie opnieuw op en probeer het opnieuw."
+"Unable to exit." = "Kan niet afsluiten."
+"Exploited kernel.\n" = "Kernel geëxploiteerd.\n"
+"Initializing jailbreak..." = "Jailbreak initialiseren..."
+"Unable to swap my process sandbox in kernel memory." = "Kan mijn processandbox niet in het kernelgeheugen omwisselen."
+"Unable to find kernel offsets." = "Kan kernel-offsets niet vinden."
+"Unable to find my process in kernel memory." = "Kan mijn proces niet vinden in het kernelgeheugen."
+"Unable to find kernel's credentials in kernel memory." = "Kan de credenties van de kernel niet vinden in het kernelgeheugen."
+"Unable to verify shenanigans in kernel memory." = "Kan shenanigans in het kernelgeheugen niet verifiëren."
+"Unable to overwrite shenanigans in kernel memory." = "Kan shenanigans in kernelgeheugen niet overschrijven."
```

Comment:
Should be "Niet in staat om shenanigans in het kernelgeheugen te overschrijven"

It's up for debate if you want to directly translate shenanigans to it's Dutch counterpart "geintjes". I'll leave that up to you.

Reply:
lol xD